guardian/csnx
# Use @nrwl:web/rollup for build

## What does this change?

Uses the [@nrwl/web:rollup](https://nx.dev/packages/web/executors/rollup#@nrwl/web:rollup) executor to build the `tsconfig` package rather than `@nrwl/js:tsc`.

Also adds a `babel.config.json` to fix [this issue](https://github.com/nrwl/nx/issues/3184#issuecomment-674770770).

Also updates all `nrwl` packages to the latest version.

I had to hardcode the asset paths rather than using `*.md` - not sure why that is. 

## How to test

Pull down the project and run `make build`. The `dist` folder should contain both `index.js` and `index.cjs` along with the other project files.
